src/doc/src/reference/specifying-dependencies.md

```diff
@@ -461,6 +461,19 @@ Like with Rust, the syntax here supports the `not`, `any`, and `all` operators
 to combine various cfg name/value pairs. Note that the `cfg` syntax has only
 been available since Cargo 0.9.0 (Rust 1.8.0).
 
+If you want to know which cfg targets are available on your platform, run
+`rustc --print=cfg` from the command line. If you want to know which `cfg`
+targets are available for another platform, such as 64-bit Windows,
+run `rustc --print=cfg --target=x86_64-pc-windows-msvc`.
+If you want to know which cfg targets are used by the release profile,
+add the `-O` parameter.
+
+This same set of values is used throughout the build for all crates in your project,
```

Comment:
I'm a little confused by how this is worded.  Instead of trying to explain _why_ it works this way, I would just be direct and say `cfg(feature = "foo")` cannot be used in the `[target]` table, with a link to the `[features]` section.  Similarly `test`, `proc-macro`, and `debug-assertions` do not work either, though I'm not sure if it is worthwhile to list everything that does or does not work.

Reply:
Not wanting to list all of the configuration options is the reason I tried to explain why it works this way. Between giving a mechanical explanation of why feature flags don't work, and pointing the user at `--print=cfg`, I figured those would allow the user to figure out which options work and which options don't.